src/components/welcome/WelcomeLogin/index.tsx

```diff
@@ -5,15 +5,13 @@ import SafeLogo from '@/public/images/logo-text.svg'
 import css from './styles.module.css'
 import { useRouter } from 'next/router'
 import WalletLogin from './WalletLogin'
-import { CREATE_SAFE_EVENTS, LOAD_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
+import { LOAD_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
 import Track from '@/components/common/Track'
-import { trackEvent } from '@/services/analytics'
 
 const WelcomeLogin = () => {
   const router = useRouter()
 
   const continueToCreation = () => {
-    trackEvent(CREATE_SAFE_EVENTS.CREATE_BUTTON)
```

Comment:
Should we still use this event or remove it altogether because there is overlap now?

Reply:
Also removing it here means we are not tracking it anymore if a user presses "Connect a wallet" on the welcome screen and gets redirected to the safe creation.

Reply:
I renamed the event to OPEN_SAFE_CREATION as it's action label is "Open stepper" and emit it here again.